app/views/declaration/summary/parties_section.scala.html

```diff
@@ -60,30 +60,55 @@
     Seq(address.fullName, address.addressLine, address.townOrCity, address.postCode, address.country).mkString("<br>")
 }
 
-@declarantEori(declarant: DeclarantDetails) = {
-    <dl class="govuk-summary-list govuk-!-margin-bottom-0">
-        <div class="govuk-summary-list__row declarant-eori-row">
-            <dt class="govuk-summary-list__key">
-                @messages("declaration.summary.parties.declarant.eori")
-            </dt>
-            <dd class="govuk-summary-list__value">
-                @declarant.details.eori.map(_.value).getOrElse("")
-            </dd>
-            <dd class="govuk-summary-list__actions"></dd>
-        </div>
-    </dl>
-}
-
 @if(sectionRequired(declarationData.parties)) {
 
     <h2 class="govuk-heading-m">Parties</h2>
 
-    @declarationData.parties.declarantDetails.map(declarant =>
-        declarantEori(declarant)
-    )
     @summaryList("declaration-parties-summary", None,
 
-        (Seq.empty :+ declarationData.parties.declarantIsExporter.map(isExporter =>
+        (Seq.empty :+ declarationData.parties.isEntryIntoDeclarantsRecords.map { isEidr =>
+            SummaryListRow(
+                classes = "is-entry-into-declarants-records-row",
+                key = Key(content = Text(messages("declaration.summary.parties.eidr"))),
+                value = Value(content = Text(messages(s"declaration.summary.parties.eidr.${isEidr.answer.toLowerCase}"))),
+                actions = Some(Actions(
+                    items = Seq(
+                        ActionItem(
+                            href = controllers.declaration.routes.EntryIntoDeclarantsRecordsController.displayPage(mode).url,
+                            content = Text(messages("site.change")),
+                            visuallyHiddenText = Some(messages("declaration.summary.parties.eidr.change"))
+                        )
+                    )
+                ))
+            )
+        })
+
+        ++ (Seq.empty :+ declarationData.parties.personPresentingGoodsDetails.map { personPresentingGoods =>
+            SummaryListRow(
+                classes = "person-presenting-goods-row",
+                key = Key(content = Text(messages("declaration.summary.parties.personPresentingGoods"))),
+                value = Value(content = Text(personPresentingGoods.eori.value)),
+                actions = Some(Actions(
+                    items = Seq(
+                        ActionItem(
+                            href = controllers.declaration.routes.PersonPresentingGoodsDetailsController.displayPage(mode).url,
+                            content = Text(messages("site.change")),
+                            visuallyHiddenText = Some(messages("declaration.summary.parties.personPresentingGoods.change"))
+                        )
+                    )
+                ))
+            )
+        })
+
+        ++ (Seq.empty :+ declarationData.parties.declarantDetails.map(declarant =>
```

Comment:
Just style comment, `map` when is bigger and contains body in new lines should have `{ }` curly brackets. More readable.

Reply:
I did not write this part. Just had to move it.